mlflow/utils/_capture_modules.py

```diff
@@ -120,6 +125,9 @@ def _load_pyfunc_patch(*args, **kwargs):
     # Store the imported modules in `output_file`
     write_to(args.output_file, "\n".join(cap_cm.imported_modules))
 
+    if spark:
+        spark.stop()
```

Comment:
use 
```
try:
  spark.stop()
catch Exception:
  pass
```
would be better, to swallow minor exceptions

Reply:
Done!